presto-main/src/main/java/com/facebook/presto/operator/scalar/ArrayFunctions.java

```diff
@@ -13,24 +13,24 @@
  */
 package com.facebook.presto.operator.scalar;
 
-import com.facebook.presto.spi.block.Block;
+//import com.facebook.presto.spi.block.Block;
```

Comment:
ok, i found the problem from before, too.  I accidentally deleted a line from the function registry.